scripts/poChecker.py

```diff
@@ -73,7 +74,7 @@ def _messageAlert(self, alert, isError=True):
 				'Translated: "{msgstr}"\n'
 				"{alertType}: {alert}"
 			.format(msgType=msgType, lineNum=self._messageLineNum,
-				msgid=self._msgid, msgstr=self._msgstr,
+				msgid=self._msgid, msgstr=self._msgstr[-1],
 				alertType="Error" if isError else "Warning", alert=alert))
```

Comment:
can this format statement be reformatted and linted please

Reply:
Done in faac8cf2d886e263f563cd8e6e895c5e2353e9d5.
I have only linted the format statement, i.e. the part that I had modified and the lines on which you have put your comment. The full statement spans over more lines however, but I have not linted the parts that I have not modified. Let me know if you prefer me to lint the whole statement though.

Reply:
Finally, I have decided to lint the whole statement anyway; that is cleaner.
See a7825844e3fb7e092f0045c65432d43f48da93db.